spec/timezone.html

```diff
@@ -424,22 +424,26 @@ <h1>CreateTemporalTimeZone ( _identifier_ [ , _newTarget_ ] )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-getisopartsfromepoch" aoid="GetISOPartsFromEpoch">
-      <h1>GetISOPartsFromEpoch ( _epochNanoseconds_ )</h1>
-      <p>
-        The abstract operation GetISOPartsFromEpoch returns the components of a date corresponding to the given number of nanoseconds since the Unix epoch in UTC.
-      </p>
+    <emu-clause id="sec-temporal-getisopartsfromepoch" type="abstract operation">
+      <h1>
+        GetISOPartsFromEpoch (
+          _epochNanoseconds_: an integer,
+        )
```

Comment:
```suggestion
        ): a Record
```
unless the record type can be even more specific